pkg/account/delete/client_test.go

```diff
@@ -117,8 +118,57 @@ func TestAccountAPIClient_DeleteGroup(t *testing.T) {
 		accountClient := delete.NewAccountAPIClient("1234", accounts.NewClient(restClient))
 
 		err = accountClient.DeleteGroup(t.Context(), "test-group")
-		assert.ErrorIs(t, err, delete.NotFoundErr)
+
+		notFoundErr := &delete.ResourceNotFoundError{}
+		assert.ErrorAs(t, err, &notFoundErr)
 	})
+
+	t.Run("successful delete of service user with no description", func(t *testing.T) {
```

Comment:
We should move this one below to the service user test. In here we have the delete group tests. We can even replace the `successful delete` test with this one

Reply:
Good catch, not sure how it ended up there 😅 . I would keep both tests for the moment.